web/src/components/navbar/NavigatorLogo.tsx

```diff
@@ -13,7 +14,7 @@ export const NavigatorLogo = (props: Props): ReactElement => {
         <img
           className={props.useSmall ? "height-3" : "height-4"}
           src="/img/Navigator-logo@2x.png"
-          alt="Business.NJ.Gov Navigator"
+          alt={`${seoTitle.title}`}
```

Comment:
```suggestion
          alt={seoTitle.title}
```

Reply:
Totally, will fix after lunch (bad habit of mine)